dashboard/db/migrate/20160315215912_create_paired_user_levels.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePairedUserLevels < ActiveRecord::Migration
+  def change
+    create_table :paired_user_levels do |t|
+      t.integer :driver_user_level_id
```

Comment:
Is there a reason to (not to?) declare the index separately from the column declaration (e.g., `t.integer :driver_user_level_id, index: true`)?